packages/sdk/src/Base.ts

```diff
@@ -437,12 +437,14 @@ class Base {
     return txOptions
   }
 
-  public getBonderAddress (token: TToken): string {
+  protected _getBonderAddress (token: TToken, sourceChain: TChain, destinationChain: TChain): string {
     token = this.toTokenModel(token)
-    return this.bonders?.[this.network]?.[token.canonicalSymbol]?.[0]
+    sourceChain = this.toChainModel(sourceChain)
+    destinationChain = this.toChainModel(destinationChain)
+    return this.bonders?.[this.network]?.[token.canonicalSymbol]?.[sourceChain.slug]?.[destinationChain.slug]
   }
 
-  public getBonderAddresses (token: TToken): string[] {
+  public getBonderAddresses (token: TToken): Routes {
```

Comment:
I loosely believe this function should be removed, as it is not used anywhere and it doesn't necessarily make sense in the context of per-route bonder, I believe. Happy to discuss otherwise.

Reply:
Agree

Resolved with https://github.com/hop-protocol/hop/commit/460cd62d1f1399a6d44661a8b355b6c688114bb2